src/utils.ts

```diff
@@ -5,9 +5,28 @@ import { JWProxy } from '@appium/base-driver';
 import type { PortForwardCallback, PortReleaseCallback } from './types';
 import type { AppiumFlutterDriver } from './driver';
 import _ from 'lodash';
+import type { StringRecord } from '@appium/types';
+import { util, node } from 'appium/support';
+import path from 'node:path';
+import fs from 'node:fs';
 
 const DEVICE_PORT_RANGE = [9000, 9020];
 const SYSTEM_PORT_RANGE = [10000, 11000];
+// Do not forget to update it if major compat changes are made
+// in the driver or server code
+const MIN_SUPPORTED_SERVER_VERSION = '0.0.15';
```

Comment:
Can we update the version to `0.0.18` which introduces the changes related to flutter-system-port argument for iOS parallel sessions.

Reply:
changed to 0.0.18
I've also moved the value to the manifest json and made it more flexible using semver